documentation/SRS.md

```diff
@@ -1,3 +1,185 @@
+
+### 1.3 Add employee
+
+#### *Main scenario*
+* User clicks button **Add**;
+* Application displays form for adding;
+* User enters information and presses **Save**;
+* If all information is valid, then information is adding in DB;
+* If don't all information is valid, then error message is displaying.
+
+![add_form](./add_employee.png)
+**pic 1.4 - View the form for adding**
+
+#### *Cancel operation scenario*
+* User clicks button **Add**;
+* Application displays form for adding;
+* User clicks button **Cancel**;
+* If user don't write information, then form will close;
+* If user writes information and don't save, 
+  then alert message is displaying.
+
+### 1.4 Edit employee
+
+#### *Main scenario*
+* User clicks button **Edit**;
+* Application displays form for editing;
+* User edits information and presses **Save**;
+* If all information is valid, then information is adding in DB;
+* If don't all information is valid, then error message is displaying.
+
+![edit_form](./edit_employee.png)
+**pic 1.5 - View the form for editing**
+
+#### *Cancel operation scenario*
+* User clicks button **Edit**;
+* Application displays form for editing;
+* User clicks button **Cancel**;
+* If user don't change information, then form will close;
+* If user change information and don't save, 
+  then alert message is displaying.
+  
+### 1.5 Delete employee
+
+#### *Main scenario*
+* User clicks button **Delete** in employee's line in lists;
+* An alert window will appear and ask if user want to delete;
+* User clicks "Yes", then information about employee
+will be deleted;
+* If user clicks "No", then window just will be closed and
+information won't be deleted.
+  
+#### *Cancel operation scenario*
+* User clicks button **Delete** in employee's line in lists;
+* An alert window will appear and ask if user want to delete;
+* User clicks "Yes", then information about employee
+will be deleted;
+![delete_form](./delete_employee.png)
+**pic 1.6 - View the form for deleting**
+  
+  
+## 2. Departments
+The mode is designed to view list of departments.
+This mode works just for edit list of departments **(pic 2.1)**.
+
+#### *Main scenario*
+* User select item Departments;
+* Application open departments list.
+
+![list_departments](./list_departments.png)
+**pic 2.1 - View the departments list**
+
+### 2.2 Add department
+
+#### *Main scenario*
+* User clicks button **Add**;
+* Application displays form for adding;
+* User enters information and presses **Save**;
+* If all information is valid, then information is adding in DB;
+* If don't all information is valid, then error message is displaying.
+
+![add_form](./add_department.png)
+**pic 2.2 - View the form for adding**
+
+#### *Cancel operation scenario*
+* User clicks button **Add**;
+* Application displays form for adding;
+* User clicks button **Cancel**;
+* If user don't write information, then form will close;
+* If user writes information and don't save, 
+  then alert message is displaying.
```

Comment:
It would be great to see a mockup for this case.

Reply:
Done